### PR TITLE
LinkResolveOperation

### PR DESCRIPTION
**Pull request by redhog**
Closes https://github.com/ucbepic/docetl/issues/108

### CODE REVIEW

File: docetl/operations/link_resolve.py [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+                schema=schema,
+                manually_fix_errors=self.manually_fix_errors,
+            )[0]
+            if self.runner.api.validate_output(self.config, output, self.console):
+                return output, True
+            return output, False
+
+        response = self.runner.api.call_llm(
+            model=self.config.get("comparison_model", self.default_model),
+            op_type="cluster",
```
Comment (shreyashankar) [inline docetl/operations/link_resolve.py:162]: maybe "resolve"?

**Comment by shreyashankar** [comment]
Thanks for this! Do you have a test or any short docs/example usage? I can add tests/docs with an example/intended usage snippet

File: docetl/operations/link_resolve.py (hunk 1 above)
Comment (redhog) [reply docetl/operations/link_resolve.py:162]: hehe, yes

**Comment by redhog** [comment]
The only data I have now is a bit too big for an example, but will try to build something minimal for the docs

**Comment by redhog** [comment]
Added a docs markdown file with an example (it's a bit stupid, but works)...

**Comment by shreyashankar** [comment]
I don't think the commit for the docs was pushed 😬

**Comment by redhog** [comment]
Because I was an idiot and pushed to the branch in my repo instead of yours  :P

**Pull request merged**